package.json

```diff
@@ -48,7 +48,7 @@
     "url-search-params-polyfill": "5.0.0",
     "uuidjs": "4.2.1",
     "webrtc-adapter": "6.4.8",
-    "wire-web-config-default": "https://github.com/wireapp/wire-web-config-default#v0.4.1"
+    "wire-web-config-default": "https://github.com/wireapp/wire-web-config-default#webapp-translations"
```

Comment:
should we make a release first?

Reply:
Yes, we will do that but first the review :)